UTIL/README.md

```diff
@@ -12,4 +12,4 @@ The CMAQ release includes several optional utilities for model developers. Chemi
 * **[inline_phot_preproc](inline_phot_preproc/README.md)**: creates photolysis reaction parameter tables for the CCTM inline photlysis module
 * **[nml](nml/README.md)**: converts chemical mechanism csv output files from chemmech to the namelist files required by the CMAQ programs
 * **[jproc](jproc/README.md)**: calculates daily look-up tables containing clear-sky photolysis rates that are used by the tabular method for calculate photolysis rates in CMAQ CCTM
-* **[wrfcmaq_twoway_coupler](wrfcmaq_twoway_coupler/README.md)**: WRF-CMAQ code to couple WRFv4.3 and CMAQv5.3.3
```

Comment:
This can be completely deleted. No wrfcmaq_twoway_coupler should be mentioned or exist (I should probably delete this entire folder...).

This code is now put directly into WRF, see [PR #1609](https://github.com/wrf-model/WRF/pull/1609).

Reply:
I deleted this line and the wrfcmaq_twoway_coupler folder.